packages/components/src/Chips/InternalChipDismissible/__tests__/InternalChipDismissible.test.tsx

```diff
@@ -199,36 +199,72 @@ describe("Deleting a chip", () => {
     );
   }
 
-  // There is a quick focus on the body before focusing on the next chip,
-  // this makes the the component difficult to test comprehensively.
-  // Strategies tried: userEvent library (typing backspace),
-  // and waitFor on the first chip's removal.
-  // eslint-disable-next-line jest/no-disabled-tests
-  it.skip("should focus on the next chip if the very first chip has focus", async () => {
+  it("should focus on the next chip if the very first chip has focus", async () => {
     render(<MockChips />);
 
     const first = getByChipLabelText("chip");
     await userEvent.tab();
     expect(first).toHaveFocus();
 
     await userEvent.keyboard("{Backspace}");
-    expect(first).not.toBeInTheDocument();
 
-    const second = getByChipLabelText("chip2");
-    expect(second).toHaveFocus();
+    // After deletion, the second chip should have focus
+    await waitFor(() => {
+      const second = getByChipLabelText("chip2");
+      expect(second).toHaveFocus();
+    });
   });
 
-  // eslint-disable-next-line jest/no-disabled-tests
-  it.skip("should focus on the previous chip if the not-first chip has focus", async () => {
+  it("should focus on the previous chip if the not-first chip has focus", async () => {
     render(<MockChips />);
 
-    const second = getByChipLabelText("chip2");
+    // Focus the second chip
     await userEvent.tab();
+    await userEvent.keyboard("{ArrowRight}");
+    const second = getByChipLabelText("chip2");
     expect(second).toHaveFocus();
+
     await userEvent.keyboard("{Backspace}");
-    expect(second).not.toBeInTheDocument();
 
-    const first = getByChipLabelText("chip2");
-    expect(first).toHaveFocus();
+    // After deletion, the first chip should have focus
+    await waitFor(() => {
+      const first = getByChipLabelText("chip");
+      expect(first).toHaveFocus();
+    });
+  });
+
+  it("should focus on the Add button when the last chip is removed", async () => {
+    function SingleChipMock() {
```

Comment:
not really necessary

we could simply delete all the chips that exist and achieve the same functionality without writing test functions that then themselves may fail or need tests of their own